config/sync/field.storage.paragraph.su_image.yml

```diff
@@ -0,0 +1,20 @@
+uuid: 2a6e844d-2e0f-4afc-9163-8994e4ff48c1
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+    - paragraphs
+id: paragraph.su_image
+field_name: su_image
```

Comment:
This name should match the pattern of the other fields. After deleting and recreating the field, remember to update the graphql and jsonapi configs.

Reply:
Updated